utils/bundles.stanza

```diff
@@ -185,6 +185,27 @@ public pcb-bundle usb-c :
   port sbu : pin[[1 2]]
   port rx : diff-pair[[1 2]]
   port tx : diff-pair[[1 2]]
+
+public pcb-bundle usb-c-full-bundle :
+  pin VBUS
+  pin CC1
+  pin CC2
+  pin DP1
+  pin DN1
+  pin DP2
+  pin DN2
+  pin SBU1
+  pin SBU2
+  pin GND
+  pin shield
+  pin SSTXP1
+  pin SSTXN1
+  pin SSRXP1
+  pin SSRXN1
+  pin SSRXP2
+  pin SSRXN2
+  pin SSTXP2
+  pin SSTXN2
```

Comment:
There is no reason to do the following:
```
port CC : pin [[1 2]]
port DP : pin [[1 2]]
```
and so on...

Reply:
That way you could refer to CC as CC[1] and CC[2]. Not sure of the utility but it does reduce the number of lines. And it allows the user to access 2 pins with a loop.

Reply:
I think I will hold off on doing this because:
1. This will break the net name extraction, unfortunately (doesn't work for array pins).
2. This will change any loop over the pins to require multiple levels of looping. It actually appears to complicate the usage of the bundle (at least in my usage).